Tools/cases_generator/generate_cases.py

```diff
@@ -273,7 +275,18 @@ def __init__(self, inst: parser.InstDef):
             else:
                 break
         self.unmoved_names = frozenset(unmoved_names)
-        if variable_used(inst, "oparg"):
+        if inst.name == 'YIELD_VALUE': breakpoint()
```

Comment:
Did you mean to make this a draft PR? :-)

Reply:
I did, actually. We need to sort out the pseudo instructions business, and a few other points.